server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java

```diff
@@ -70,6 +70,8 @@
 
   private final boolean enableUnseqCompaction = IoTDBDescriptor.getInstance().getConfig()
       .isEnableUnseqCompaction();
+  private final boolean enableContinuousCompaction = IoTDBDescriptor.getInstance().getConfig()
```

Comment:
remove the field as the field is never used in this file

Reply:
ok